src/apiCalls.js

```diff
@@ -0,0 +1,23 @@
+const fetchAllMovies = (path) => {
+        return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${path}`)
+        .then(response => {
+        if (!response.ok) {
+            throw new Error('Error with requesting movies.');
+          } else {
+           return response.json();
+          }
+        })
+}
+
+ const fetchSingleMovie = (id) => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+    .then(response => {
+    if (!response.ok) {
+        throw new Error('Error with requesting movies.');
+      } else {
+       return response.json();
+      }
+    })
+}
+
+export { fetchAllMovies, fetchSingleMovie }
```

Comment:
These Fetch calls look remarkably similar - instead of having two, you could refactor these into one by sending through the entire url, rather than just a portion of it. Thru this would could use one fetch function for all movies _and_ for a specific one. Let me know if you have any questions about this.